### PR TITLE
Update Parcel tests

### LINKED ISSUE #409: Small update for Parcel
~(These are in the nightly version, not sure if that's a problem...)~

- https://bundlers.tooling.report/non-js-resources/css/source/
	- The linked issue is fixed. Sourcemaps aren't emitted anymore for inline assets.
- https://bundlers.tooling.report/non-js-resources/css/url/
	- The linked issue is fixed, importing CSS as a URL works correctly now.
- https://bundlers.tooling.report/transformations/dead-code-dynamic/
	- https://v2.parceljs.org/features/code-splitting/#unused-exports

### PR DESCRIPTION
**Pull request by mischnic**
Closes #409

cc @surma

### CODE REVIEW

**Comment by mischnic** [comment]
@googlebot I signed it!

File: tests/non-js-resources/subtests/css/subtests/source/parcel/index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-result: partial
+result: pass
 issue:
   - url: https://github.com/parcel-bundler/parcel/issues/4320
     fixedSince: 2.0.0-beta.1
     status: closed
   - url: https://github.com/parcel-bundler/parcel/issues/4855
+    fixedSince: 2.0.0-beta.1
```
Comment (surma) [inline tests/non-js-resources/subtests/css/subtests/source/parcel/index.md]: Did you men beta2 here or was it actually fixed in beta1 and we didn’t test it right?

**Review by surma — CHANGES_REQUESTED** [review]
Thanks for providing the update, @mischnic! I appreciate it! 

Would you be so kind and also update the test’s individual `package.json` to depend on `beta.2` where necessary?

File: tests/non-js-resources/subtests/css/subtests/source/parcel/index.md (hunk 1 above)
Comment (mischnic) [reply tests/non-js-resources/subtests/css/subtests/source/parcel/index.md]: Oops, you're right

**Comment by mischnic** [comment]
@surma Done!

**Review by surma — APPROVED** [review]
Cheers!

**Pull request merged**